src/collectd.conf.pod

```diff
@@ -8635,6 +8636,10 @@ Rather than using the local time when dispatching a value, read the timestamp
 from the field with the zero-based index I<Index>. The value is interpreted as
 seconds since epoch. The value is parsed as a double and may be factional.
 
+=item B<LastLineOnly> I<true>|I<false>
+
+Rather than sending all lines in the CSV file, we only enter the last one
```

Comment:
IMHO, the below proposed text would be more clear:
"If set to true, only the last line in the CSV file will be read. If set to false (the default), all lines will be read."